docs/scos/dev/technical-enhancement-integration-guides/integrating-symfony-6.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Integrating Symfony 6
+description: Learn about the main changes in the new Symfony version 6
+last_updated: Dec 7, 2022
+template: howto-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/symfony-6-integration
+originalArticleId: d5e96c3b-3ed6-49ed-982c-aa641e09b559
+redirect_from:
+  - /2021080/docs/symfony-6-integration
+  - /2021080/docs/en/symfony-6-integration
+  - /docs/symfony-6-integration
+  - /docs/en/symfony-6-integration
+  - /v6/docs/symfony-6-integration
+  - /v6/docs/en/symfony-6-integration
+  - /docs/scos/dev/technical-enhancements/symfony-6-integration.html
```

Comment:
Note for the TW, please pay attention to those redirects, they should be removed I guess. CC @AlexSlawinski

Reply:
I can remove them, sure. Is that what you want @asmarovydlo ?

Reply:
Yes, that is what I think we should do. But please confirm with someone from TW team.